systemtest/src/test/resources/tracing/1.25/jaeger-instance.yaml

```diff
@@ -10,6 +10,8 @@ spec:
       log-level: debug
       query:
         base-path: /jaeger
+      collector.grpc.tls.enabled: false
+      reporter.grpc.tls.enabled: false
```

Comment:
Shouldn't we consider using newer versions of Jaeger? Of course not as part of this PR.

Reply:
I can create an issue for that if you agree :)